apps/native-component-list/src/screens/FontScreen.tsx

```diff
@@ -183,3 +200,50 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
   },
 });
+
+const size = 150;
+
+function useLoadIcon(getImage: () => Promise<RenderToImageResult | null>) {
+  const [icon, setIcon] = useState<RenderToImageResult | null>(null);
+  useEffect(() => {
+    const loadIcon = async () => {
+      const icon = await getImage();
+      console.log({ icon });
+      if (icon) {
+        setIcon(icon);
+      } else {
+        console.error('Failed to load icon');
+      }
+    };
+    loadIcon();
+  }, []);
+  return icon;
+}
+
+function VectorIconSection() {
+  const icon1 = useLoadIcon(() => MaterialIcons.getImageSource('camera', size, 'yellow'));
+  const icon2 = useLoadIcon(() => FontAwesome.getImageSource('book', size, 'yellow'));
+  const icon3 = useLoadIcon(() => Ionicons.getImageSource('camera', size, 'yellow'));
+  const icon4 = useLoadIcon(() => FoundationIcons.getImageSource('book', size, 'yellow'));
+  const icons = [icon1, icon2, icon3, icon4];
```

Comment:
The `getImageSource` is broken here

Reply:
removed these changes, I'll need to push an update to expo/vector-icons